test/unit/Tabs.test.js

```diff
@@ -177,12 +177,12 @@ describe('Tabs handle Tab selection logic', () => {
     });
     test('Tab can be clicked and will display its content', () => {
         tabs.find('[value="tab-2"]').simulate('click');
-        const renderedContent = tabs.find('.tab-content > div').html();
+        const renderedContent = tabs.find(Tab).html();
         expect(renderedContent).toEqual('<div>Tab 2 child</div>');
     });
     test('Tab without value will still be clickable', () => {
         tabs.find('[value="tab-2"]').simulate('click');
-        const renderedContent = tabs.find('.tab-content > div').html();
+        const renderedContent = tabs.find(Tab).html();
```

Comment:
This test makes no sense. It's the exact copy of the one above with a different name. It's clearly not testing that case.. removing.